nodejs/nodejs-cloud-run-hello-world/cloudbuild.yaml

```diff
@@ -0,0 +1,76 @@
+steps:
+
+- id: 'Code Style'
+  name: 'node:12-slim'
+  entrypoint: '/bin/bash'
+  args:
+  - '-c'
+  - |
+    npm install
+    npm run lint
+
```

Comment:
Why do you need this step? Building docker image based on Dockerfile should handle this.

Reply:
I switched to running tests via JS. That requires dev dependencies. We don't install them inside the container currently, we could switch to running them inside but then the container is further from production ready.

I can remove this "Code Style" step, I added it because I had convenient linting scripts in the sample and needed the node_modules installed for the integration step.